dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/service/SoftwareService.java

```diff
@@ -36,82 +38,84 @@
    * Constants.
    */
   private static final ObjectMapper MAPPER = new ObjectMapper();
+  private static final SoftwareProperties config = new SoftwareProperties();
 
   /**
    * Configuration.
    */
-  private String mavenRepositoryUrl = "https://artifacts.oicr.on.ca/artifactory";
 
   @SneakyThrows
-  public List<ArtifactFolder> getVersions(String artifactId, SoftwareProperties config) {
+  public List<ArtifactFolder> getVersions(String artifactId) {
     String versionsUrl;
-    versionsUrl = mavenRepositoryUrl + "/api/storage/" + config.getIcgcRepository() + "/" +
+    versionsUrl = config.getMavenRepositoryUrl() + "/api/storage/" + config.getIcgcRepository() + "/" +
         artifactId;
     val url = new URL(versionsUrl);
     val response = MAPPER.readValue(url, ArtifactFolderResults.class);
     return response.children;
   }
 
   @SneakyThrows
-  public List<MavenArtifactVersion> getMavenVersions(String artifactId, SoftwareProperties config) {
-    val versionsUrl = mavenRepositoryUrl + "/api/search/versions?g=" + config.getGroupId() + "&a=" +
+  public List<MavenArtifactVersion> getMavenVersions(String artifactId) {
+    val versionsUrl = config.getMavenRepositoryUrl() + "/api/search/versions?g=" + config.getGroupId() + "&a=" +
         artifactId + "&repos=" + config.getRepository();
     val url = new URL(versionsUrl);
     val response = MAPPER.readValue(url, MavenArtifactResults.class);
     return response.results;
   }
 
-  public String getLatestVersionUrl(SoftwareProperties config) {
-    return getVersionUrl("%5BRELEASE%5D", config.getArtifactId(), config);
+  public String getLatestVersionUrl() {
+    return getVersionUrl("%5BRELEASE%5D", config.getArtifactId());
   }
 
-  public String getLatestICGCGetVersionUrl(String os, SoftwareProperties config) {
-    List<ArtifactFolder> result = getVersions("icgc-get", config);
-
-    return getICGCGetVersionUrl("0.2.10", os, config);
+  public String getLatestICGCGetVersionUrl(String os) {
+    List<ArtifactFolder> result = getVersions(config.getIcgcArtifactId());
```

Comment:
Please use `val` on the left hand side of all of these statements. Reduces type noise.